Data collection (metode afsnit) (#70)
chhoumann/p8: report/src/system_design/data_collection_design.tex

\section{Phase one: Data collection}\label{sec:knn_implementation}
As the quality of the system is reliant on the accuracy and precision of the map during this phase it is critical to create a map of high quality.
Therefore, steps must be taken to ensure this quality. 
\citeauthor{improving_indoor_localization}~\cite{improving_indoor_localization} suggests that measurements be taken by systematically partitioning the area in a grid and then performing measurements for each cell of the grid for n-seconds. 
We propose a design where, instead of having a fixed collection time, a fixed minimum number of measurements is received from each beacon.
We do this, as beacons broadcast on a fixed time-schedule \cite{apple2023ibeacon}.
This is done to ensure that all beacons have a representative set of measurements with low standard deviation when constructing the map.
To ensure a low standard deviation, we use the following procedure.

> Data is collected from each beacon until a minimum number of data points are collected, and a  satisfiable standard deviation is reached.

Reviewer: ```suggestion
Data is collected from each beacon until a minimum number of data points are collected, and a satisfiable standard deviation is reached.
```

> Data is collected from each beacon until a minimum number of data points are collected, and a  satisfiable standard deviation is reached.

Reviewer 2: OK?

```suggestion
Data is collected from each beacon until a minimum number of data points are collected, and a stable standard deviation within a satisfactory threshold is reached.
```

Reviewer 2: Or just 
```suggestion
Data is collected from each beacon until a minimum number of data points are collected, and a satisfactory standard deviation is reached.
```
